packages/runtime/src/runtime.ts

```diff
@@ -580,6 +632,30 @@ export class Runtime {
     }
   }
 
+  // transfer ASSET as per transaction parameters
+  transferAsset (txnParam: AssetTransferParam): void {
+    const fromAssetHolding = this.getAssetHolding(txnParam.assetID, txnParam.fromAccount.addr);
+    const toAssetHolding = this.getAssetHolding(txnParam.assetID, txnParam.toAccountAddr);
+
+    this.assertAssetNotFrozen(txnParam.assetID, txnParam.fromAccount.addr);
```

Comment:
does it mean `to` can receive assets even if it is frozen ? could you please confirm ?

Reply:
It's probably both. But i can't test it to be 100% sure, as algob doesn't support asset freeze right now. Will validate after we add this support.
```
An Asset Freeze Transaction is issued by the Freeze Address and results in the asset receiver address losing or being granted the ability to send or receive the frozen asset.
```
Adding assert for `to` as well.